src/all/pandachaika/src/eu/kanade/tachiyomi/extension/all/pandachaika/PandaChaikaDto.kt

```diff
@@ -44,7 +44,7 @@ class LongArchive(
     private val filecount: Int,
     private val filesize: Double,
     private val tags: List<String>,
-    private val title_jpn: String?,
+    private val title_jpn: String = "",
```

Comment:
I'm not sure if this is necessary if you use `title_jpn?.takeIf` in `toManga()`. Up to you if you want to change it back.